Week3/homework/4-step3.js

```diff
@@ -1,2 +1,11 @@
 'use strict';
-// paste your freeCodeCamp solutions in here
+
+//constructor, Dog, with properties name, color, and numLegs that are set to a string, a string, and a number, respectively.
+function Dog() {
+    this.name = 'Buki';
+    this.color = 'Blue';
+    this.numLegs = 4;
+
+}
+//create a new instance of Dog, assigning it to a variable hound
+let hound = new Dog();
```

Comment:
Good, this follows the brief.
However, what if I want another dog, this one called Stella, who's white and with 4 legs?
As a generic definition of a Dog, this falls down. Here is another way to approach it:

function Dog(name, colour, numLegs) {
  this.name = name;
  this.colour = colour;
  this.numLegs = numLegs;
}

const stella = new Dog('Stella', 'white', 4);
const holly = new Dog('Holly', 'caramel', 4);
const tucker = new Dog('Tucker', 'Cinnamon and white', 4);

console.log(stella, holly, tucker);
Then we can reuse Dog for each example of a dog.